docker/smqtk_iqr_playground/default_confs/runApp.IqrClassifier.json

```diff
@@ -0,0 +1,6 @@
+{
+    "classifier": {
+        "smqtk_classifier.impls.classify_descriptor_supervised.sklearn_logistic_regression.SkLearnLogisticRegression": {},
+        "type": "smqtk_classifier.impls.classify_descriptor_supervised.sklearn_logistic_regression.SkLearnLogisticRegression"
+    }
+}
```

Comment:
This config is I think for an unrelated service. Did you mean to bring this file in?

Reply:
Yeah, I brought that in for the `iqrTrainClassifier` script since it looked like that was part of the original web demo. Should I remove that section from the demo?

Reply:
OK, I won't get too picky, we can leave it in for now.